### PR TITLE
Add gio to tests

### PR DESCRIPTION
**Pull request by EPashkin**
5 errors

### CODE REVIEW

**Comment by EPashkin** [comment]
No namespaces in records, and `void`

**Comment by EPashkin** [comment]
Fixed void*

**Comment by EPashkin** [comment]
About array, we can add it to `use glib_ffi::{..` but IMHO must be better fix

**Comment by gkoz** [comment]
We'll still want a `test` crate that depends on others because it makes sense to add some small libraries that aren't `gtk` dependencies (`libsecret`, `libvte`, other `g*` ones), and that crate could also do some sanity checks (ideally, it would compare Rust and C `sizeof` for some types).

**Comment by gkoz** [comment]
I'll have to think about the `Array` types, didn't expect them.

**Comment by EPashkin** [comment]
Ok, now i understand, agreed with that.

**Comment by EPashkin** [comment]
I add `sys_test` module with empty lib.rs here while you thinking.
May be better add array fix to master and rebase current.

**Comment by EPashkin** [comment]
Ok, i change fix void*

**Comment by gkoz** [comment]
> Ok, i change fix void*

Not just yet... :)

**Comment by EPashkin** [comment]
Added crate, moved fix void* to last

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@ target
 /src/version.rs
 tests/gir-files
 tests/sys/*-sys/**
+tests/sys/sys_build/**
 !tests/sys/*-sys/Cargo.toml
```
Comment (gkoz) [inline .gitignore]: This line probably has to become
```
!tests/sys/*/Cargo.toml
```
then.

Comment (EPashkin) [inline .gitignore]: Not needed: git show changes in added files

Comment (gkoz) [inline .gitignore]: No, why even ignore the whole crate if you only have to ignore its `Cargo.lock` and `target`.

Comment (gkoz) [inline .gitignore]: I mean this crate isn't autogenerated after all.

**Comment by EPashkin** [comment]
Updated .gitignore, ffi_type.rs

**Comment by gkoz** [comment]
I think I'll merge this to work on the arrays with more coverage.

**Comment by gkoz** [comment]
Thanks

**Pull request merged**